.konflux/Dockerfile.catalog

```diff
@@ -1,5 +1,5 @@
 # The opm image is expected to contain /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
-ARG OPM_IMAGE=registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.19
+ARG OPM_IMAGE=registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.20
```

Comment:
@shajmakh  looks like we will need to use the brew registry for now : `brew.registry.redhat.io/rh-osbs/openshift-ose-operator-registry-rhel9:v4.20`
until 4.20 is released.